app/presenters/facet/range_presenter.rb

```diff
@@ -0,0 +1,41 @@
+module Facet
+  class RangePresenter < FacetPresenter
+    def display(options = {})
```

Comment:
Unused method argument - `options`. If it's necessary, use `_` or `_options` as an argument name to indicate that it won't be used. You can also write as `display(*)` if you want the method to accept any arguments but don't care about them.<br>Assignment Branch Condition size for display is too high. [37.34/15]<br>Method has too many lines. [35/10]